GPyOpt/acquisitions/NEI.py

```diff
@@ -0,0 +1,131 @@
+from GPyOpt.acquisitions.base import AcquisitionBase
+from GPyOpt.core.task.cost import constant_cost_withGradients
+from ..util.general import get_quantiles
+from ..models.gpmodel import GPModel
+from torch.quasirandom import SobolEngine
+from scipy.stats import norm
+import numpy as np
+import operator
+import functools
+from time import time
+# from profilehooks import profile
+
+class AcquisitionNEI(AcquisitionBase):
+    
+    """
+    Noisy Expected Improvement with quasi-monte carlo integration acquisition funciton.
+
+    :param model: GPyOpt class of model
+    :param space: GPyOpt class of domain
+    :param optimizer: optimizer of the acquisition. Should be a GPyOpt optimizer
+    :param cost_withGradients: function that provides the evaluation cost and its gradients
+
+    """
+
+    # --- Set this line to true if analytical gradients are available
+    analytical_gradient_prediction = True
+
+    
+    def __init__(self, model, space, optimizer, cost_withGradients=None, N_QMC = None,jitter = 0, opt_restarts=1):
+
+        self.optimizer = optimizer
+        self.N_QMC = N_QMC
+        self.jitter=jitter
+        self.batch_elements = []
+        self.opt_restarts = opt_restarts
+        super(AcquisitionNEI, self).__init__(model, space, optimizer)
+        
+        # --- UNCOMMENT ONE OF THE TWO NEXT BITS
+             
+        # 1) THIS ONE IF THE EVALUATION COSTS MAKES SENSE
+        #
+        # if cost_withGradients == None:
+        #     self.cost_withGradients = constant_cost_withGradients
+        # else:
+        #     self.cost_withGradients = cost_withGradients 
+
+        # 2) THIS ONE IF THE EVALUATION COSTS DOES NOT MAKE SENSE
+        #
```

Comment:
What's that for? Users won't be able to do it

Reply:
Ye sorry it's just a comment in the template for a new acquisition function, and I forgot to remove it

Reply:
I actually just realized that the cost_withGradients is taken care of in the initialization of the AcquisitionBase class.. so i can just remove all of it